content/docker-hub/configure-sign-in.md

```diff
@@ -5,29 +5,229 @@ keywords: authentication, registry.json, configure,
 title: Enforce sign-in for Desktop
 ---
 
-By default, members of your organization can use Docker Desktop on their machines without signing in to any Docker account. To ensure that a user signs in to a Docker account that is a member of your organization and that the
-organization’s settings apply to the user’s session, you can use a `registry.json` file.
+By default, members of your organization can use Docker Desktop without signing
+in. When users don’t sign in as a member of your organization, they don’t
+receive the [benefits of your organization’s
+subscription](../subscription/details.md) and they can circumvent [Docker’s
+security features](../desktop/hardened-desktop/_index.md) for your organization.
 
-The `registry.json` file is a configuration file that allows administrators to specify the Docker organization the user must belong to and ensure that the organization’s settings apply to the user’s session. The Docker Desktop installer can create this file on the users’ machines as part of the installation process.
+To ensure members of your organization always sign in, you can deploy a
+**`registry.json`** configuration file to the machines of your users.
```

Comment:
Copy-pasted this from notion without noticing that some occurrences got bolded. Will fix.
```suggestion
`registry.json` configuration file to the machines of your users.
```